_episodes/01-introduction.md

```diff
@@ -244,12 +286,13 @@ print(energy_kJ)
 ## Making choices: logic Statements
 Within your code, you may need to evaluate a variable and then do something if the variable has a particular value.  This type of logic is handled by an `if` statement.  In the following example, we only append the negative numbers to a new list.  
```

Comment:
Is this the first time append is introduced?

Reply:
No, `append` is introduced with `for` loops in the previous section. See `Line 200` in the md file.